packages/models-library/src/models_library/api_schemas_webserver/socketio.py

```diff
@@ -0,0 +1,17 @@
+from ..users import GroupID, UserID
+
+
+class SocketIORoom(str):
+    __slots__ = ()
+
+    @classmethod
+    def from_socket_id(cls, socket_id: str) -> "SocketIORoom":
+        return cls(socket_id)
+
+    @classmethod
+    def from_group_id(cls, group_id: GroupID) -> "SocketIORoom":
+        return cls(group_id)
```

Comment:
question: why not then group:{group_id} ?

Reply:
In theory we could and should look better